include/alpaka/math/cbrt/CbrtStdLib.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Axel Huebl, Benjamin Worpitz
+/* Copyright 2022 Axel Huebl, Benjamin Worpitz, Bernhard Manfred Gruber
```

Comment:
This doesn't seem right, the file was last touched in 2021.

Reply:
Seems I accidentially included this while testing the commands. Sorry!